v21.1/cockroach-debug-list-files.md

```diff
@@ -0,0 +1,81 @@
+---
+title: cockroach debug list-files
+summary: Learn the command for listing the files collected in the debug zip.
+toc: true
+key: debug-list-files.html
+---
+
+The `cockroach debug list-files` [command](cockroach-commands.html) shows the files that will be collected by using [`cockroach debug zip`](cockroach-debug-zip.html).
+
+{{site.data.alerts.callout_info}}
+The files listed include logs, heap profiles, and goroutines. Other [files](cockroach-debug-zip.html#files) generated by `cockroach debug zip` are not listed by `cockroach debug list-files`.
+{{site.data.alerts.end}}
+
+## Subcommands
+
+{% include {{ page.version.version }}/misc/debug-subcommands.md %}
+
+## Synopsis
+
+~~~ shell
+$ cockroach debug list-files {flags}
+~~~
+
+## Flags
+
+The `debug list-files` subcommand supports the following [general-use](#general), [client connection](#client-connection), and [logging](#logging) flags.
+
+### General
+
+Flag | Description
+-----|-----------
+`--cert-principal-map` | A comma-separated list of `<cert-principal>:<db-principal>` mappings. This allows mapping the principal in a cert to a DB principal such as `node` or `root` or any SQL user. This is intended for use in situations where the certificate management system places restrictions on the `Subject.CommonName` or `SubjectAlternateName` fields in the certificate (e.g., disallowing a `CommonName` like `node` or `root`). If multiple mappings are provided for the same `<cert-principal>`, the last one specified in the list takes precedence. A principal not specified in the map is passed through as-is via the identity function. A cert is allowed to authenticate a DB principal if the DB principal name is contained in the mapped `CommonName` or DNS-type `SubjectAlternateName` fields.
```

Comment:
`--host`, `--certs-dir`, `--cluster-name`, etc are the generic client options. Is there a way to share them with the other client commands like `cockroach init`?

It would be nice to have this table highlights the flags that are unique to this command.